site/content/en/guides/consumer/function/export/circleci/_index.md

```diff
@@ -72,9 +72,9 @@ version: "2.1"
 
 To merge the exported file with your existing pipeline, you can:
 
-1.  Copy and paste the `orbs` field
-1.  Insert a `checkout` step as the first step in the `run-functions` job.
-1.  Add `kpt/run-functions` to your workflow jobs.
+1. Copy and paste the `orbs` field
```

Comment:
Should all items have the number `1`?

Reply:
Yes. "To be practical and avoid errors on the numbers, use "1" for all the items. The markdown engine will output them in the correct order." from https://about.gitlab.com/handbook/markdown-guide/#ordered-lists